src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,26 @@
 package racingcar;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import racingcar.domain.Judgement;
+import racingcar.domain.Racingprocess;
+import racingcar.domain.Userconsole;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+        Judgement referee = new Judgement();
+        Userconsole ask = new Userconsole();
+        Racingprocess race = new Racingprocess();
+
+        ArrayList<String> cars = ask.makingCarList();
+        int rounds = ask.askRounds();
```

Comment:
```suggestion
        int rounds = userConsole.readRounds();
```
변수 이름은 일반적으로 명사 혹은 명사구로 작성해요! 위처럼 작성하면 문장을 읽듯이 잘 읽히지 않나요?
이와 관련해서는 [표현적 가독성, 기능적 가독성에 관한 글](https://medium.com/naver-cloud-platform/%EB%84%A4%EC%9D%B4%EB%B2%84%ED%81%B4%EB%9D%BC%EC%9A%B0%EB%93%9C-%EA%B0%9C%EB%B0%9C%EC%9E%90-%EC%8A%A4%ED%86%A0%EB%A6%AC-%EC%A2%8B%EC%9D%80-%EC%BD%94%EB%93%9C%EB%9E%80-%EB%AC%B4%EC%97%87%EC%9D%BC%EA%B9%8C-%ED%81%B4%EB%A6%B0%EC%BD%94%EB%93%9C-%EC%9D%B4%EC%95%BC%EA%B8%B0-c7811f73a46b) 을 읽어보셔도 좋을 것 같아 링크 남깁니다!

Reply:
참고할 글 감사합니다. 코딩 컨벤션에 대해서 중요성도 깨닫고 좀더 인식하면서 작성하게 된것 같습니다.